Document/0x06a-Platform-Overview.md

```diff
@@ -129,7 +129,7 @@ A language.lproj folder exists for each language that the application supports.
 
 ![OWASP MSTG](Images/Chapters/0x06a/iOS_project_folder.png) \
 
-On a jailbroken device, you can recover the IPA for an installed iOS app using different tools that allow decrypting the main app binary and reconstruct the IPA file. Similarly, on a jailbroken device you can install the IPA file with [IPA Installer](https://github.com/autopear/ipainstaller "IPA Installer"). During mobile security assessments, developers often give you the IPA directly. They can send you the actual file or provide access to the development-specific distribution platform they use, e.g., [HockeyApp](https://hockeyapp.net/ "HockeyApp") or [TestFlight](https://developer.apple.com/testflight/ "TestFlight").
+On a jailbroken device, you can recover the IPA for an installed iOS app using different tools that allow decrypting the main app binary and reconstruct the IPA file. Similarly, on a jailbroken device you can install the IPA file with [IPA Installer](https://github.com/autopear/ipainstaller "IPA Installer"). During mobile security assessments, developers often give you the IPA directly. They can send you the actual file or provide access to the development-specific distribution platform they use, e.g. [TestFlight](https://developer.apple.com/testflight/ "TestFlight").

```

Comment:
@cpholguera  Do we want to replace hockeyapp with appcenter https://appcenter.ms

refer: https://devblogs.microsoft.com/appcenter/hockeyapp-is-being-retired/

Reply:
That's a great idea, could you please edit the sentence again on a new PR?

Reply:
Let me add that in.

Reply:
https://github.com/OWASP/owasp-mstg/pull/1963 created